servers/zms/src/main/java/com/yahoo/athenz/zms/DBService.java

```diff
@@ -7994,6 +8033,117 @@ public DomainList listDomainDependencies(String service) {
         }
     }
 
+    void executeDeleteExpiredMemberships(ResourceContext ctx, ObjectStoreConnection con, int roleId, String roleName,
```

Comment:
We should return to executeDeleteExpiredMembership since we're processing only a single member and not members